lib/ts/utils.ts

```diff
@@ -122,3 +122,5 @@ export function validateTheStructureOfUserInput(config: any, inputSchema: any, c
 export function frontendHasInterceptor(req: BaseRequest): boolean {
     return getRIDFromRequest(req) !== undefined;
 }
+
+export let clientIdsForDevRecipe: string[] = [];
```

Comment:
Add comment here explaining what this is used for. Also, name is terrible. Call it `thirdPartyProvidersClientIds`